server/server.mjs

```diff
@@ -12,24 +12,23 @@ import { connectDB } from "./config/connection.mjs";
 // };
 
 const Express = async () => {
-  return { express }; //? added await to the return value 4.22.2024
+  return new { express }(); //? added await to the return value 4.22.2024
```

Comment:
injected new into imports to create a child of the import instance asynchronously